python/pyspark/sql/pandas/serializers.py

```diff
@@ -175,6 +178,16 @@ def wrap_and_init_stream():
         return super(ArrowStreamUDFSerializer, self).dump_stream(wrap_and_init_stream(), stream)
 
 
+class ArrowStreamUDTFSerializer(ArrowStreamUDFSerializer):
+    """
+    Same as :class:`ArrowStreamSerializer` but it flattens the struct to Arrow record batch
+    for applying each function with the raw record arrow batch. See also `DataFrame.mapInArrow`.
+    """
+
+    def load_stream(self, stream):
+        return super(ArrowStreamUDFSerializer, self).load_stream(stream)
```

Comment:
This may be confusing. How about directly call it?
```suggestion
        return super(ArrowStreamUDFSerializer, self).load_stream(stream)
        return ArrowStreamSerializer.load_stream(self, stream)
```

Reply:
SGTM